raiden-dapp/src/locales/en.json

```diff
@@ -792,10 +793,12 @@
       },
       "injected-provider": {
         "header": "Injected Provider",
+        "description": "Connect to the provider of your web-browser possibly added with an installed plugin.",
         "error-message": "Failed to link to provider. Please make sure to grant the requested permissions."
       },
       "wallet-connect-provider": {
         "header": "WalletConnect",
+        "description": "Connect to any mobile wallet app on your phone that supports linking with WalletConnect.",
```

Comment:
"Connect with any mobile wallet app that supports WalletConnect"

Reply:
✔️